scripts/convert_tiny_autoencoder_to_diffusers.py

```diff
@@ -0,0 +1,72 @@
+import argparse
+
+from diffusers.utils import is_safetensors_available
+
+
+if is_safetensors_available():
+    import safetensors.torch
+else:
+    raise ImportError("Please install `safetensors`.")
+
+from diffusers import TinyAutoencoder
+
+
+"""
+Example - From the diffusers root directory:
+
+Download the weights:
+```sh
+$ wget -q https://huggingface.co/madebyollin/taesd/resolve/main/taesd_encoder.safetensors
+$ wget -q https://huggingface.co/madebyollin/taesd/resolve/main/taesd_decoder.safetensors
```

Comment:
FYI, I will be AFK for a few days and won't be able to check `huggingface.co/madebyollin` PRs until end of the week

Reply:
I don't think it will be blocker. 

Thanks for your awesome work, you rock!